projects/exui-common-lib/src/lib/components/selected-case/selected-case.component.spec.ts

```diff
@@ -2,19 +2,26 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { of } from 'rxjs';
 import { SharedCase } from '../../models/case-share.model';
 import { UserDetails } from '../../models/user-details.model';
+import { FeatureToggleService } from '../../services';
 import { SelectedCaseComponent } from './selected-case.component';
 
 describe('SelectedCaseComponent', () => {
   let component: SelectedCaseComponent;
   let fixture: ComponentFixture<SelectedCaseComponent>;
   let user: UserDetails;
   let shareCases: SharedCase[] = [];
+  const mockFeatureToggleService = jasmine.createSpyObj('FeatureToggleService', ['getValue']);
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ SelectedCaseComponent ]
+      declarations: [ SelectedCaseComponent ],
+      providers: [{
+        provide: FeatureToggleService,
+        useValue: mockFeatureToggleService
+      }]
     })
     .compileComponents();
+    mockFeatureToggleService.getValue.and.returnValue(of(true));
```

Comment:
we need to abstract this flag out as discussed in our design approach in following the twelve factor principle for web development..we cant be having to rebuild the application each time we need to do a change...please move out from the app into a config which we can drive through our custom config which are examples in code already.

Reply:
moved the flag to webapp as per other comments.